src/XMakeCommandLine/CommandLineSwitches.cs

```diff
@@ -438,14 +438,22 @@ bool unquoteParameters
             bool parametersStored = false;
 
             // if this is the first time this switch has been detected
-            if (_parameterizedSwitches[(int)parameterizedSwitch].commandLineArg == null)
+            if (_parameterizedSwitches[(int) parameterizedSwitch].commandLineArg == null)
             {
                 // initialize its parameter storage
-                _parameterizedSwitches[(int)parameterizedSwitch].parameters = new ArrayList();
-            }
+                _parameterizedSwitches[(int) parameterizedSwitch].parameters = new ArrayList();
```

Comment:
Can you change the spacing around casting to int back, please? You can run codeformatter but in this case it should just be four space deletions.